.circleci/config.yml

```diff
@@ -254,6 +255,10 @@ jobs:
 
               while [ ! -f ~/miniconda/pytorch_installed ]; do sleep 2; done # wait for Pytorch
               pytest
+
+              #re-install with bullet and run physics tests again
+              python setup.py install --headless --bullet
```

Comment:
This should actually be `./build.sh` as the tests will only pickup in-tree bindings and don't work with the setup.py install method (grumble grumble grumble)

Reply:
Odd that the tests were passing after the rebuild, then. Do you think they were picking up the previous build?

Reply:
Yeah, they were picking up things from the previous invocation of `build.sh`

Reply:
:expressionless: